src/app/pages/ChainSettings/manage-url-entities-view/create-modal.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { memo, useCallback, useState } from 'react';
+
+import { Controller, useForm } from 'react-hook-form';
```

Comment:
pls try to use the latest version of react-hook-form in all new ui flows, so we have fewer places to fix in the future. You can install it alongside old version like showed [here](https://github.com/madfish-solutions/templewallet-extension/pull/1189/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R175).